packages/manager/modules/pci/src/projects/project/storages/object-storage/users/users.controller.js

```diff
@@ -75,4 +75,20 @@ export default class PciStoragesContainersUsersController {
         ),
       );
   }
+
+  downloadRcloneFile(user) {
+    this.downloadOpenStackRclone(user);
+  }
+
+  showSecretKey(user) {
+    this.CucCloudMessage.success(
+      this.$translate.instant(
+        'pci_projects_project_storages_containers_users_show_secret_key_success',
+        {
+          user: user.username,
+          secret: user.s3Credentials[0].secret,
```

Comment:
**Question**: Are we always going to need the first element from `s3Credetials` array?

Reply:
Yes